### PR TITLE
New back storages for atmospheric technicians (this time, it's real)

### PR DESCRIPTION
**Pull request by KingFroozy**
## About the PR

PR about adding new back storages for atmospheric technicians.

Atmos techs now have unique backpacks, satchels, and duffel bags, with it's own textures and descriptions.

These storages are accessable through an AtmosDrobe, And techies themselves will roundstart with them.

ATTENTION: I'm not really good at speaking english, so descriptions might be a little wrong, please let me know, if you'll find any mistakes in it.

P. S. - Finally, I understood, how to work in github and change text in files :D

**Media**

AtmoBacks
- [ ] I have added screenshots/videos to this PR showcasing its changes ingame.

**Changelog**

:cl:
- add: Added new backpacks, satchels, and duffel bags for atmospheric technicians.

### CODE REVIEW

**Comment by potato1234x** [comment]
you forgot to credit yourself in the meta.jsons

**Comment by potato1234x** [comment]
also 23 commits is way too many just do like 1 for something like this :)

File: Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/backpack.yml [hunk 1]
```diff
@@ -1,313 +1,322 @@
...
+      - id: BoxSurvival
+
+- type: entity
+  parent: ClothingBackpackAtmos
+  id: ClothingBackpackAtmosFilled
+  noSpawn: true
+  components:
+  - type: StorageFill
+    contents:
+      - id: BoxSurvival
```
Comment (Emisse) [inline Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/backpack.yml:17]:
```suggestion
      - id: BoxSurvivalEngineering

```

File: Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/duffelbag.yml [hunk 2]
```diff
@@ -1,225 +1,235 @@
...
+      - id: BoxSurvival
+
+- type: entity
+  noSpawn: true
+  parent: ClothingBackpackDuffelAtmos
+  id: ClothingBackpackDuffelAtmosFilled
+  components:
+  - type: StorageFill
+    contents:
+      - id: BoxSurvival
```
Comment (Emisse) [inline Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/duffelbag.yml:17]:
```suggestion
      - id: BoxSurvivalEngineering

```

File: Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/satchel.yml [hunk 3]
```diff
@@ -1,234 +1,244 @@
...
+      - id: BoxSurvival
+
+- type: entity
+  noSpawn: true
+  parent: ClothingBackpackSatchelAtmos
+  id: ClothingBackpackSatchelAtmosFilled
+  components:
+  - type: StorageFill
+    contents:
+      - id: BoxSurvival
```
Comment (Emisse) [inline Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/satchel.yml:17]:
```suggestion
      - id: BoxSurvivalEngineering

```

File: Resources/Prototypes/Entities/Clothing/Back/backpacks.yml [hunk 4]
```diff
@@ -1,258 +1,287 @@
...
+  - type: UserInterface
+    interfaces:
+    - key: enum.StorageUiKey.Key
+      type: StorageBoundUserInterface
+  # to prevent bag open/honk spam
+  - type: UseDelay
+    delay: 0.5
+
+- type: entity
+  parent: Clothing
```
Comment (Emisse) [inline Resources/Prototypes/Entities/Clothing/Back/backpacks.yml:31]: make this parent off ClothingBackpack so you can delete all the unnecessary lines of code

File: Resources/Prototypes/Entities/Clothing/Back/duffel.yml [hunk 5]
```diff
@@ -1,234 +1,266 @@
...
+  - type: UserInterface
+    interfaces:
+    - key: enum.StorageUiKey.Key
+      type: StorageBoundUserInterface
+  # to prevent bag open/honk spam
+  - type: UseDelay
+    delay: 0.5
+
+- type: entity
+  parent: Clothing
```
Comment (Emisse) [inline Resources/Prototypes/Entities/Clothing/Back/duffel.yml:34]: same this is why we use parents

File: Resources/Prototypes/Entities/Clothing/Back/satchel.yml [hunk 6]
```diff
@@ -1,173 +1,202 @@
...
-    sprite: Clothing/Back/Satchels/holding.rsi
-    state: icon
-    layers:
-    - state: icon
-    - state: icon-unlit
-      shader: unshaded
-  - type: Storage
-    capacity: 9999
+- type: entity
+  parent: Clothing
```
Comment (Emisse) [inline Resources/Prototypes/Entities/Clothing/Back/satchel.yml:2]: same

**Comment by Emisse** [comment]
diffs also looking super cooked on this one chief lines of code changed should be only the ones you touched what ide are you using

**Comment by potato1234x** [comment]
it should be something like Modified from tgstation blah blah blah by KingFroozy (Github) for SS14

**Comment by potato1234x** [comment]
if it would be less work you could just use my pr

File: Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/backpack.yml [hunk 7]
```diff
@@ -1,313 +1,323 @@
...
+
+- type: entity
+  parent: ClothingBackpackAtmos
+  id: ClothingBackpackAtmosFilled
+  noSpawn: true
+  components:
+  - type: StorageFill
+    contents:
+      - id: BoxSurvivalEngineering
```
Comment (lzk228) [inline Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/backpack.yml:17]: bruh. just put it at the and of file, so diff would be not so big, and same in duffelbag.yml and satchel.yml

File: Resources/Prototypes/Entities/Clothing/Back/backpacks.yml [hunk 8]
```diff
@@ -1,258 +1,287 @@
...
+    containers:
+      storagebase: !type:Container
+        ents: []
+  - type: UserInterface
+    interfaces:
+    - key: enum.StorageUiKey.Key
+      type: StorageBoundUserInterface
+  # to prevent bag open/honk spam
+  - type: UseDelay
+    delay: 0.5
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Clothing/Back/backpacks.yml:57]: and here too, out it in the end

**Review by lzk228 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**